Makefile

```diff
@@ -1,5 +1,9 @@
 prefix ?= /usr
 
+SOURCE_DATE_EPOCH ?= $(shell git log -1 --pretty=%ct)
+# https://reproducible-builds.org/docs/archives/
```

Comment:
:heart: 

This was a good read, I especially like this bit, given our recent discussions:

`Care must be taken to ensure that sort is called in the context of the C locale to avoid any surprises related to collation order.`

Reply:
Right; the doc mentions that GNU tar's `--sort=name` is always locale insensitive.